baybe/searchspace/discrete.py

```diff
@@ -704,9 +718,10 @@ def _apply_pandas_constraint_filter(
     return df
 
 
-def _apply_polars_constraint_filter(
-    ldf: pl.LazyFrame, constraints: Collection[DiscreteConstraint]
-) -> pl.LazyFrame:
+def _apply_constraint_filter_polars(
+    ldf: pl.LazyFrame,
+    constraints: Collection[DiscreteConstraint],
```

Comment:
Still needs to be sequence
```suggestion
    constraints: Sequence[DiscreteConstraint],
```

Reply:
amended last commit